ci/jjb/jobs/pulp3-smash-runner.yaml

```diff
@@ -97,6 +103,22 @@
             git clone https://github.com/pulp/pulp_rpm.git --branch master
             git clone https://github.com/pulp/pulp_docker.git --branch master
 
+            # Configure podman for docker tests 
+            sudo yum install -y podman
+            sudo bash -c 'cat << EOF > /etc/containers/registries.conf
+            [registries.search]
+            registries = ["docker.io", "registry.fedoraproject.org", "quay.io", "registry.access.redhat.com", "registry.centos.org"]
+
+            [registries.insecure]
+            registries = ["${PULP_SMASH_SYSTEM_HOSTNAME}:8080"]
```

Comment:
This line is not evaluating, see `/job/pulp3-smash-runner/83/consoleFull`

I suspect this is not evaluating because the use of `bash -c` creates a subshell where envvars are not defined.

Need to change it or try using `| tee`